src/components/TemplateLibrary/index.js

```diff
@@ -137,7 +126,10 @@ const LibraryComponent = () => {
       </pkg:part>
       ${spec}
       </pkg:package>`;
-      context.document.body.insertOoxml(ooxml, Word.InsertLocation.end);
+      const range = context.document.body.insertOoxml(ooxml, Word.InsertLocation.end);
+      const control = range.insertContentControl();
```

Comment:
Change the variable name to `contentControl`.

Reply:
Okay Sure :)